src/components/views/elements/AppTile.tsx

```diff
@@ -322,7 +324,13 @@ export default class AppTile extends React.Component<IProps, IState> {
             switch (payload.action) {
                 case 'm.sticker':
                     if (this.sgWidget.widgetApi.hasCapability(MatrixCapabilities.StickerSending)) {
-                        dis.dispatch({ action: 'post_sticker_message', data: payload.data });
+                        dis.dispatch({
+                            action: 'post_sticker_message',
+                            data: {
+                                threadId: this.props.threadId,
+                                ...payload.data,
+                            },
```

Comment:
Is it intentional that the Widget could specify a different threadId? IMO L330 and L331 should be swapped